debezium-connector-ibmi/src/test/java/io/debezium/connector/db2as400/util/TestHelper.java

```diff
@@ -16,12 +16,14 @@
 
 public class TestHelper {
 
-    private static final String DATABASE_NAME = "DTEST";
+    private static final String DATABASE_NAME = "AZIZACALM2";
```

Comment:
There should be no changes in this class. These are just defaults and if you need to use different values then they should be passed via system properties from outside during runtime. OTH thinking about that it makes sense to keep HOTNAME/PORT change to point to PUB400 as the default.